junit5/src/main/java/sunstone/core/JUnit5Config.java

```diff
@@ -0,0 +1,21 @@
+package sunstone.core;
+
+
+public class JUnit5Config {
```

Comment:
I can use https://github.com/wildfly-extras/sunstone/blob/master/core/src/main/java/org/wildfly/extras/sunstone/api/impl/Config.java properties. For Azure ARM it would work, I would have `junit5.azure-arm.password` which is OK. `EC2` needs to be renamed to `AWS`. I would also suggest moving credentials properties into separate class.

JCloud stack is not supposed to be used with JUnit5 so I am not worried about duplicates in `sunstone.properties`

Reply:
As we spoke, this comment is invalid - unifying the credentials as I intended goes against the original intent of multiple providers per cloud. Resolving.